### PR TITLE
Impl onLoggedOut, update onLoggedIn to only add data.extra non-empty obj

### PR DESCRIPTION
**Pull request by stevenhornung**
- Implemented the onLoggedOut hook for logout
- Updated the onLoggedIn hook to only append data.extra when a non-empty object was returned
- added tests for onLoggedOut hook in new file
- moved onLoggedIn hook tests to same file as onLoggedOut hook tests
- Updated README to no longer say '(coming soon)' for onLoggedOut hook data.extra

### CODE REVIEW

**Comment by kahmali** [comment]
Looking good! I'm a bit busy today but I'll review this later tonight. If you don't hear back from me by tomorrow come back here and yell at me :) Thanks so much for taking care of all of this. 

The one thing that jumps out to me that you'll probably think I'm crazy for is the commit messages. They are all slightly out of [spec](https://github.com/kahmali/meteor-restivus/blob/devel/CONTRIBUTING.md#commit-messages). The first needs to be shortened up, the second should use the imperative tone, and the 2nd and 3rd both need sentence capitalization. Just reword/rebase those and force push to this PR branch (otherwise you have to create a new PR – your choice). I do accept subject lines up to 80 chars, since those display properly everywhere in GitHub. I need to update the guidelines to include that asap.

Just know that I'm this crazy [even to members of the Meteor Developer Group](https://github.com/kahmali/meteor-restivus/pull/46#issuecomment-98184151) :) Nobody is exempt from my git OCD :P

Thanks again for what appears to be an awesome PR. I'll get this reviewed and merged in ASAP.

**Comment by stevenhornung** [comment]
@kahmali must have missed this as a notification! I've created a new PR with a single commit (with better/shorter wording :) ).

I'm going to close this PR.
